### DISCUSSION TITLE
bmp280 error ????

### DISCUSSION THREAD

**Discussion by icelon** [micropython/micropython#9287] [Libraries & Drivers]
Hello, a few days ago I received some bmp280 modules and the truth is, I don't know why they don't work.
for now it is connected, it has push resistors and it is a 3.3v device
```
Scan i2c bus...
i2c devices found: 1
Decimal address:  118  | Hexa address:  0x76
```
but when trying to capture the data

```
I2C scan result :  [118]
I2C connection successfull
Traceback (most recent call last):
  File "<stdin>", line 22, in <module>
  File "/lib/bmp280.py", line 97, in __init__
  File "/lib/bmp280.py", line 126, in _read
OSError: [Errno 5] EIO
```

**Comment by robert-hh** [comment]
Does the library use the address 118 (or 0x76)? It may by default use 0x77.
